### PR TITLE
Release notes for ringpop-go v0.6.0.

### PR DESCRIPTION
**Pull request by Motiejus**
Let's do a release!

### CODE REVIEW

**Comment by CorgiMan** [comment]
Looks good!

File: CHANGES.md [hunk 1]
```diff
@@ -3,8 +3,20 @@ ringpop-go changes
 
 Next Release
 ------------
+
+v0.6.0
+------------
+
+* Fix: add 1-buffer for error channels, prevents leaking goroutines. #140
+* Feature: support headers forwarding. #141
```
Comment (mennopruijssers) [inline CHANGES.md]: should be "support head forwarding" or "support forwarding headers"?

File: CHANGES.md [hunk 2]
```diff
@@ -3,8 +3,20 @@ ringpop-go changes
...
 * Feature: testpop takes an additional -stats-udp or -stats-file flag that can be used to emit
   statsd messages to a file or over the network. #144
+* Fix: race condition when loading hostlist in discover provider. #145
+* Testing: add suspect, faulty and tombstope period command-line flags. #148
```
Comment (mennopruijssers) [inline CHANGES.md]: `... to testpop`?.

File: CHANGES.md [hunk 3]
```diff
@@ -3,8 +3,20 @@ ringpop-go changes
 
 Next Release
```
Comment (mennopruijssers) [inline CHANGES.md]: Should we remove this section when merging to master?

**Comment by mennopruijssers** [comment]
Some small nits, but LGTM!

**Pull request merged**